packages/adapters/wagmi/src/connectors/AuthConnector.ts

```diff
@@ -157,6 +158,7 @@ export function authConnector(parameters: AuthParameters) {
       if (!this.provider) {
         this.provider = W3mFrameProviderSingleton.getInstance({
           projectId: parameters.options.projectId,
+          chainId: ChainController.getActiveCaipNetwork()?.caipNetworkId,
```

Comment:
are there other usages we need to add the param to?

Reply:
All seem have now